src/main/java/fi/hsl/transitdata/pulsarpubtransconnect/PubtransTableHandler.java

```diff
@@ -133,77 +142,48 @@ public void setLastModifiedTimeStamp(long ts) {
         return commonBuilder.build();
     }
 
-
-    class JourneyInfo {
-        String direction;
-        String routeName;
-        String startTime;
-        String operatingDay;
-
-        JourneyInfo(Map<String, String> map) {
-            direction = map.get(TransitdataProperties.KEY_DIRECTION);
-            routeName = map.get(TransitdataProperties.KEY_ROUTE_NAME);
-            startTime = map.get(TransitdataProperties.KEY_START_TIME);
-            operatingDay = map.get(TransitdataProperties.KEY_OPERATING_DAY);
-            if (!isValid()) {
-                //Let's print more info for debugging purposes:
-                log.warn("JourneyInfo is missing some fields. Content: " + this.toString());
+    protected Optional<PubtransTableProtos.DOITripInfo> getTripInfo(long dvjId, long jppId) {
+        try {
+            String stopIdKey = TransitdataProperties.REDIS_PREFIX_JPP + Long.toString(jppId);
+            Optional<String> maybeStopId = Optional.ofNullable(jedis.get(stopIdKey));
```

Comment:
Could these redis calls be wrapped?

Reply:
What are JPP and DVJ btw?

Reply:
JourneyPatternPoint, is it perhaps a unique stop identifier? dunno, @laaksma ? dvj-id is the trip id.

Reply:
not sure if I understand the redis wrapping...? to some class / interface?

Reply:
Yes, utility functions. Although not necessary if redis is not used elsewhere?

Reply:
good idea, done.